grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -708,6 +726,10 @@ public GrpcService build() {
             throw new IllegalStateException(
                     "'unframedGrpcErrorHandler' can only be set if unframed requests are enabled");
         }
+        if (!enableUnframedRequests  && unframedGrpcStatusFunction != null) {
```

Comment:
How about combining this condition with the above condition?
```java
if (!enableUnframedRequests) {
    if (unframedGrpcErrorHandler != null || unframedGrpcStatusMapper != null) {
        ...
    }
}
```

Reply:
But then can't we print error messages differently for invalid `unframedGrpcErrorHandler` or `unframedGrpcStatusMapper` setting cases?

Reply:
Or, how about like the below..?

```java
if (!enableUnframedRequests) {
    if (unframedGrpcErrorHandler != null) {
        ...
    }
    if (unframedGrpcStatusMapper != null) {
        ...
    }
}
```

Reply:
> Or, how about like the below..?

That also works. 😄